src/main/java/edu/uc/cech/agrawadv/eventorganizer/resource/Resource.java

```diff
@@ -10,12 +10,15 @@
 @Entity
 @Table(name = "resource")
 public class Resource {
+	
```

Comment:
I added spacing just because it was a practice we used at my last co-op. Each company has their own but it has been instilled in me to make sure each line has space above it and space below it when necessary.